external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

```diff
@@ -374,8 +380,15 @@ class KafkaTestUtils extends Logging {
 
     def shutdown() {
       factory.shutdown()
-      Utils.deleteRecursively(snapshotDir)
-      Utils.deleteRecursively(logDir)
+      // The directories are not closed even if the ZooKeeper server is shut-downed.
+      // Please see ZOOKEEPER-1844, which is fixed in 3.4.6+. It leads to test failures
+      // on Windows if these are not ignored.
+      if (FileUtils.deleteQuietly(snapshotDir)) {
```

Comment:
Do you mean to check if it's _not_ deleted? but I think you can just use the `Utils.deleteRecursively` method here